feat: update organization from nf-core to genomic medicine sweden (#35)
genomic-medicine-sweden/autoseq: CHANGELOG.md

- [ #19 ](https://github.com/imsarath/nf-autoseq/pull/19) Repo setup for Genome Medicine Sweden Org
- [ #21 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/21) Added sub-workflow for tumor biomarker profiling - purecn-run and typeDPYD

### `Changed`

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Updated all repository references from `nf-core/autoseq` to `genomic-medicine-sweden/autoseq` across configs, docs, assets, workflows, and pipeline files.

Reviewer: ```suggestion
- [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Refactor repository name from `nf-core/autoseq` to `genomic-medicine-sweden/autoseq`.
```

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Expanded `CONTRIBUTING.md` with detailed PR title conventions, review guidelines, and GMS-specific contribution workflow.

Reviewer: This should have been a different PR

Author: Fair point, I bundled these (bug_report template and contributing guidelines) since they were all part of the initial GMS migration cleanup, but I agree that separate PRs would have been cleaner. I'll keep that in mind for next time.

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Updated bug report and issue templates to remove nf-core Slack references and align with GMS organisation.

Reviewer: same here

Author: Responded to a similar comment above.

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Updated `README.md` contributors section.

Reviewer 2: These three could in my opinion have been seperate PRs

Author: I see your point, they do cover different areas (docs vs. templates). I bundled them since they were all related to the migration to the GMS organization. If it's okay, I'd prefer to keep them together this time to get them merged, but I'll make sure to keep PRs more granular in the future.

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Updated `README.md` contributors section.

Reviewer: same here

Author: Responded to a similar comment above.

### `Fixed`

> - [ #35 ](https://github.com/genomic-medicine-sweden/nf-autoseq/pull/35) Rebuilt `nextflow_schema.json` to pass `nf-core schema build` validation after the organisation rename.

Reviewer 2: Nice!

Author: Thanks :+1:
